ksql-cli/src/main/java/io/confluent/ksql/Ksql.java

```diff
@@ -74,25 +74,24 @@ public static void main(final String[] args) throws IOException {
     }
   }
 
-  @SuppressFBWarnings(
-      value = "OBL_UNSATISFIED_OBLIGATION",
-      justification = "'input' already in try with resource block - i.e. false positive")
   @SuppressWarnings("OptionalUsedAsFieldOrParameterType")
   private static Properties loadProperties(final Optional<String> propertiesFile) {
     final Properties properties = new Properties();
-    propertiesFile.ifPresent(file -> {
-      try (FileInputStream input = new FileInputStream(file)) {
-        properties.load(input);
-        if (properties.containsKey(KsqlConfig.KSQL_SERVICE_ID_CONFIG)) {
-          properties.put(
-              StreamsConfig.APPLICATION_ID_CONFIG,
-              properties.getProperty(KsqlConfig.KSQL_SERVICE_ID_CONFIG)
-          );
-        }
-      } catch (final IOException e) {
-        throw new KsqlException("failed to load properties file: " + file, e);
+    if (!propertiesFile.isPresent()) {
+      return properties;
+    }
+
+    try (InputStream input = new FileInputStream(propertiesFile.get())) {
+      properties.load(input);
+      if (properties.containsKey(KsqlConfig.KSQL_SERVICE_ID_CONFIG)) {
```

Comment:
This can be removed from here. CLI should not send `StreamsConfig.APPLICATION_ID_CONFIG` or `KsqlConfig.KSQL_SERVICE_ID_CONFIG` anyway.

Reply:
That may be the case, but this PR is very small and targeted at fixing just the FB issue. Next time ;)